pkg/delete/pointer/delete_pointer.go

```diff
@@ -36,6 +37,9 @@ type DeletePointer struct {
 
 	// ActionType and Domain are used when deleting key-user-actions-web entities
 	ActionType, Domain string
+
+	//OriginID is DT ID of the configuration. Mutually exclusive with Identifier.
+	OriginID string
```

Comment:
nit: i think we call this "OriginObjectId" everywhere else.

Reply:
I'll change this with the PR for the document delete

Reply:
done